README.md

```diff
@@ -7,7 +7,7 @@
 3. Authenticate with `yarn graph auth https://api.thegraph.com/deploy/ <access-token>`
```

Comment:
Any idea why this might be happening? 
`yarn run deploy:base`

<img width="1319" alt="image" src="https://github.com/ScopeLift/umbra-subgraph/assets/61768337/0916c06b-ddf9-49b0-9ea8-6c8bd3d9d4b0">

Reply:
I Have `BASE_DEPLOY_KEY` set in my `.env`

Reply:
After some troubleshooting, we determined that the steps for deploy after a "yarn install" require the "prepare" step to be done before the "build" and "codegen" steps.